pkg/rancher-desktop/assets/specs/command-api.yaml

```diff
@@ -208,6 +208,47 @@ paths:
         '400':
           description: An error occurred
 
+  /v1/port_forwarding:
+    post:
+      operationId: createPortForward
+      summary: Create a new port forwarding
+      requestBody:
+        description: JSON block consisting of the port forwarding details
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                namespace:
+                  type: string
+                service:
+                  type: string
+                k8sPort:
+                  type: string
+                hostPort:
+                  type: integer
+        required: true
+      responses:
+        '200':
+          description: The port forwarding was created.
+        '400':
+          description: The port forwarding could not be created.
+    delete:
+      operationId: deletePortForward
+      summary: Delete a port forwarding
+      parameters:
+      - in: query
```

Comment:
Shouldn't the parameters get schemata, and possibly descriptions?

Reply:
I'm not sure what you mean. I followed the syntax other usages in this file for query parameters which only provide the `name`.

Reply:
Yeah, we can do that as a follow-up and fix everything at once. (Currently we don't describe the fact that namespace is a string, for example.)